packages/cf-funding-protocol-contracts/manual-deploy.js

```diff
@@ -11,6 +11,8 @@ const getContractAddress = ethers.utils.getContractAddress;
 const provider = new InfuraProvider("rinkeby", process.env.INFURA_API_KEY);
 const wallet = Wallet.fromMnemonic(process.env.ETH_ACCOUNT_MNENOMIC).connect(provider);
 
+console.log(wallet.address);
```

Comment:
Remove?

Reply:
easier to check the pending transactions when its sent from the wallet

Reply:
Makes sense - there wasn't a comment or anything around it so I thought it was for debugging purposes mostly